client/jetpack-cloud/sections/settings/advanced-credentials/host-selection/index.tsx

```diff
@@ -74,7 +74,7 @@ const HostSelection: FunctionComponent = () => {
 
 	const recordHostSelectionEvent = ( host: string ) => {
 		dispatch(
-			recordTracksEvent( 'jetpack_advanced_credentials_flow_host_select', {
+			recordTracksEvent( 'calypso_jetpack_advanced_credentials_flow_host_select', {
```

Comment:
This event name triggers an error in Calypso. It seems ok to append `calypso_`, since no other event in the codebase starts with `jetpack`, including events triggered from cloud.